src/server_manager/install_scripts/do_install_server.sh

```diff
@@ -174,3 +175,5 @@ done
 # Wait for install script to finish, so that if there is any error in install_server.sh,
 # the finish trap in this file will be able to access its error code.
 wait $install_pid
+curl -sSL https://agent.digitalocean.com/install.sh | sh
```

Comment:
Let's add a little commentary here, e.g.:
```sh
wait $install_pid

# Install the DigitalOcean Agent, for improved monitoring:
# https://www.digitalocean.com/docs/monitoring/quickstart/#enable-the-digitalocean-agent-on-existing-droplets
# 
# Since the server manager looks only for the tags created in the previous
# step, this does not slow down server creation.
curl -sSL https://agent.digitalocean.com/install.sh | sh
```